test/tests/val-2a/tests

```diff
@@ -0,0 +1,18 @@
+[Tests]
+  design = 'EquilibriumBC.md'
```

Comment:
You are not using the EquilibriumBC, make sure you only list objects that you use.